File: Aula1-06102021/teste-variaveis.py
# 08/10/2021 - Estudo Algoritmos Aula 01 - Variáveis - Python
# Referência: https://www.w3schools.com/python/python_variables.asp

x = 5
y = "Luiz"

# alterando o tipo da variável após ela já ter sido declarada
x = "Carlos"

print(x)
print(y)

# casting em python:

x = str(3)    # x será '3'
y = int(3)    # y será 3
z = float(3)  # z será 3.0 

print(x)
print(y)
print(z)

# verificando os tipos em Python - função type():

print(type(x))
print(type(y))
print(type(z))

# strings podem ser declaradas utilizando aspas simples ou dupla:

t = "da Silva"
w = 'da Silva'

print(t)
print(w)
print(type(t))
print(type(w))

# os nomes das variaveis são case-sensitive:

g = 4
G = "Joana"

print(g)

File: Aula1-06102021/testevariaveis.c
/* 08/10/2021 - Estudo Algoritmos Aula 01 - Variáveis - C
Referência: https://www.tutorialspoint.com/cprogramming/c_variables.htm */

#include <stdio.h>

//Declaracao de variaveis

extern int a,b;
extern int c;
extern float f;
//extern double d;

int main () {
	
	/* definicao das variaveis */
	int a,b;
	int c;
	float f;
	//double d;
	
	/* inicializacao real */
	a = 10;
	b = 20;
	
	c = a+b;
	printf("o valor de c é: %d \n",c);
	
	f = 70.0/3.0;
	printf("o valor de f é: %f \n",f);
	
	return 0;
}
	
